Platform/CoffeelakeBoardPkg/CfgData/CfgData_Gpio.dsc

```diff
@@ -49,248 +49,249 @@
-# !BSF SUBT:{GPIO_TMPL:GPP_J05: 0x03502385: 0x89052019}
-# !BSF SUBT:{GPIO_TMPL:GPP_J06: 0x03502385: 0x89062001}
-# !BSF SUBT:{GPIO_TMPL:GPP_J07: 0x03502385: 0x89072019}
-# !BSF SUBT:{GPIO_TMPL:GPP_J08: 0x03502381: 0x89082001}
-# !BSF SUBT:{GPIO_TMPL:GPP_J09: 0x03502281: 0x89092001}
-# !BSF SUBT:{GPIO_TMPL:GPP_J10: 0x0500E2E1: 0x090A0001}
-# !BSF SUBT:{GPIO_TMPL:GPP_J11: 0x03502383: 0x890B2009}
-# !BSF SUBT:{GPIO_TMPL:GPP_K00: 0x050005E1: 0x0A000001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K01: 0x03506281: 0x8A012017}
-# !BSF SUBT:{GPIO_TMPL:GPP_K02: 0x050005E1: 0x0A020001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K03: 0x05148DA1: 0x0A030619}
-# !BSF SUBT:{GPIO_TMPL:GPP_K04: 0x050022E1: 0x0A040001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K05: 0x050005E1: 0x0A050001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K06: 0x050005E1: 0x0A060201}
-# !BSF SUBT:{GPIO_TMPL:GPP_K07: 0x050005E1: 0x0A070001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K08: 0x050062E1: 0x0A080001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K09: 0x03502383: 0x8A092001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K10: 0x03506281: 0x8A0A2001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K11: 0x03502383: 0x8A0B2001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K12: 0x05348DA1: 0x0A0C0601}
-# !BSF SUBT:{GPIO_TMPL:GPP_K13: 0x050062E1: 0x0A0D0001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K14: 0x030022E1: 0x0A0E0001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K15: 0x030022E1: 0x0A0F0001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K16: 0x050022E1: 0x0A100001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K17: 0x050062E1: 0x0A110001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K18: 0x050005E1: 0x0A120001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K19: 0x03502381: 0x8A132001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K20: 0x050005E1: 0x0A140001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K21: 0x050005E1: 0x0A150001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K22: 0x05506281: 0x8A162001}
-# !BSF SUBT:{GPIO_TMPL:GPP_K23: 0x050062E1: 0x0A170001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d00: 0x07502383: 0x8B002001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d01: 0x07502383: 0x8B01201F}
-# !BSF SUBT:{GPIO_TMPL:GPP_d02: 0x07502383: 0x8B02201F}
-# !BSF SUBT:{GPIO_TMPL:GPP_d03: 0x07502383: 0x8B032019}
-# !BSF SUBT:{GPIO_TMPL:GPP_d04: 0x07502283: 0x8B042001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d05: 0x07502283: 0x8B052001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d06: 0x07502283: 0x8B062001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d07: 0x07502281: 0x8B072001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d08: 0x07502383: 0x8B082001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d09: 0x07502283: 0x8B092001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d10: 0x07502283: 0x8B0A2001}
-# !BSF SUBT:{GPIO_TMPL:GPP_d11: 0x07502283: 0x8B0B2001}
-# !BSF SUBT:{GPIO_TMPL:GPP_PEC: 0x00000000: 0x80000000}
+#                               DW0       : DW1      :               Native Function 1                    :                   Native Function 2               :                   Native Function 3               : Native Function 4
+# !BSF SUBT:{GPIO_TMPL:GPP_A00: 0x03502381: 0x80002001: RCIN#                                             : PCH_H = N/A   PCH_LP = TIME_SYNC1                 : PCH_H = ESPI_ALERT1#   PCH_LP = N/A               : PCH_H = N/A}
```

Comment:
In Native function 4, there is only PCH_H = N/A. What about PCH_LP?

Reply:
PCH_LP doesnt have Native Function 4 at all. So listed only PCH_H there.

Reply:
Got it. Thanks.